File-Formats/DocIO/Working-with-Charts/Format-Legend.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Modify the appearance of legend | Syncfusion
+description: Learn how to modify the appearance of legend in a chart in a  Word document using Syncfusion .NET Core Word (DocIO) library without Microsoft Word.
```

Comment:
description: Learn how to modify the appearance of legend in a chart in a Word document using Syncfusion .NET Core Word (DocIO) library without Microsoft Word.